api/cotrip_app/models.py

```diff
@@ -1,3 +1,28 @@
 from django.db import models
 
+
+class User(models.Model):
```

Comment:
Django actually comes with a User model already. You can use that one instead of creating your own.